docs/sample_project_structure/README.md

```diff
@@ -0,0 +1,96 @@
+# Name of Sample <!-- omit in toc -->
+
+One line blurb about this sample.
+
+## Contents <!-- omit in toc -->
+
+- [Solution Overview](#solution-overview)
+  - [Background](#background)
+  - [Architecture](#architecture)
+  - [Technologies used](#technologies-used)
+- [Key Learnings](#key-learnings)
+  - [1. Key Learnings 1](#1-key-learnings-1)
+  - [2. Key Learnings 2](#2-key-learnings-2)
+- [Key Concepts](#key-concepts)
+  - [\[EXAMPLE\] Continuous Integration](#example-continuous-integration)
+  - [\[EXAMPLE\] Continuous Delivery](#example-continuous-delivery)
+- [How to use the sample](#how-to-use-the-sample)
+  - [Prerequisites](#prerequisites)
+  - [Setup and Deployment](#setup-and-deployment)
+    - [Deployed Resources](#deployed-resources)
+- [Known Issues, Limitations and Workarounds](#known-issues-limitations-and-workarounds)
+
+---------------------
+
+## Solution Overview
+
+### Background
+
+Here is the background of the sample.
+
+### Architecture
+
+The following is the overall architecture of the sample.
+
+### Technologies used
+
+This sample uses:
+
+- [Azure Technology](www.microsoft.com)
+
+## Key Learnings
+
+The following are Key Learnings in this sample:
+
+### 1. Key Learnings 1
+
+### 2. Key Learnings 2
+
+## Key Concepts
+
+Engineering concepts demonstrated.
+
+### [EXAMPLE] Continuous Integration
+
+### [EXAMPLE] Continuous Delivery
+
+## How to use the sample
+
+### Prerequisites
+
+1. [EXAMPLE] [Github account](https://github.com/)
+
+#### Software pre-requisites if you don't use dev container<!-- omit in toc -->
+
+- [EXAMPLE] [az cli x.y+](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+
+#### Software pre-requisites if you use dev container<!-- omit in toc -->
+
+*EXAMPLE:*
+
+- [Docker](https://www.docker.com/)
+- [VSCode](https://code.visualstudio.com/)
+- [Visual Studio Code Remote Development Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+
+1. Rename `.envtemplate` to `devcontainer.env` and update the values as explained in the **Initial Setup** section.
+2. Open the project inside the vscode dev container.
+3. `cd` into the `e2e/<name-sample>` or `single-tech/<name-of-technology>` folder and run `./deploy.sh` inside the dev container terminal.
+
+### Setup and Deployment
+
+> IMPORTANT NOTE: As with all Azure Deployments, this will **incur associated costs**. Remember to teardown all related resources after use to avoid unnecessary costs. See [here](#deployed-resources) for a list of deployed resources.
```

Comment:
```suggestion
> IMPORTANT NOTE: As with all Azure deployments, this will **incur associated costs**. Remember to teardown all related resources after use to avoid unnecessary costs. See [here](#deployed-resources) for a list of deployed resources.
```